### DISCUSSION TITLE
Mac?

### DISCUSSION THREAD

**Discussion by Imjudah** [redeclipse/base#1512] [Help and Support] [answered]
I notice there's a mac version on the website, but not the steam version. Why is that?

**Comment by dc-redeclipse** [comment]
Hallo and welcome to Red Eclipse!

Not at the moment but on intel you can use Proton and on Apple Silicon, you can use Whiskey it's running fine as well.

**Comment by dc-redeclipse** [comment]
Maybe I will play with the Game Porting Toolkit from apple a bit to find a better solution for that.

**Comment by Jigoku** [comment]
> I notice there's a mac version on the website, but not the steam version. Why is that?

As for native support, we've had to drop mac unfortunately due to Apple deprecating OpenGL in favour of Metal. 

You will now have to use something like proton/whisky as mentioned above 👌(i can't vouch for how well it works with RE).

**Comment by 20kdc** [comment]
Sorry if this is weird, but as a (non-Mac-using-or-owning) developer of projects that target Mac I'd like to know what https://github.com/redeclipse/base/commit/848d3d05a78f54a03b284decb5226324eb40ddbb "Apple walled garden becomes mandatory in April, removing all macOS support" is referring to.

**Reply by dc-redeclipse** [reply]
Since Apple makes building for macOS harder, we just give it up at the moment. Also, the user base for macOS is very, very small at the moment and good ways to go around are exiting as mentioned above. But I will try with Apple's Game Porting Toolkit to make a release working again for macOS out of the Box soon this year. Its experimental so far.

**Comment by qreeves** [comment, marked as answer]
For future reference, Apple now requires all software to be signed, which requires US$99 per year for a developer account, there was a grace period being offered by Valve and Apple where it'd run apps under Steam's credentials but that is now over. I can't justify paying for a Mac and a developer account subscription and all the extra work to keep things duct taped together for 0.2% of total installs.

I was gonna yank Mac support prior to v1.6 because I saw the writing on the wall then, but I let people away my decision with sunk cost fallacies. It took me weeks to fix the retina display thing because it super samples and reports everything incorrectly back to the program. This isn't the first or last time they will break things and make us fix it until we give up and die. Apple thinks it knows better than everyone else.

Even if someone paid for the subscription and the Mac there's no amount of money you can pay me personally to make me put up with that again. MAC SUPPORT IS DISCONTINUED. Blame Apple, not us.

**Reply by MirceaKitsune** [reply]
That is so unbelievably stupid of them, but nothing I wouldn't expect from a company like Apple. I definitely wouldn't waste money on such a thing, especially as much as 100$ an year! I presume you technically can still create software that runs on Apple just not host it in their official store and have it easy to install, though recent changes made even that hard, though now more than ever I think the case for having Apple support third party app stores should be made. It's really not our fault at this point, large tech companies are just out to destroy themselves and any good thing they have control over with them.
